nav2_behavior_tree/include/nav2_behavior_tree/bt_service_node.hpp

```diff
@@ -230,9 +229,9 @@ class BtServiceNode : public BT::ActionNodeBase
   void increment_recovery_count()
   {
     int recovery_count = 0;
-    config().blackboard->template get<int>("number_recoveries", recovery_count);  // NOLINT
+    [[maybe_unused]] auto res = config().blackboard->get("number_recoveries", recovery_count);  // NOLINT
```

Comment:
it is `[[no_discard]]` for a reason!!! Basically, it may segfault, if things don't go as planned.
 
Here I am working around the lack of safety, when using this interface. I am not endorsing it, just trying to postpone a better alternative to future PRs.